### PR TITLE
Document language format

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
@hodgestar Ready for review.

File: docs/example.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+   type: choice
+   save: colour
+
+   question[multiple-choice]:`
+     Hi {@msisdn}. What is your favourite colour?
+     1. Red {$red}
+     2. Blue {$blue}
+   `
+   routes:
+     red: chose-red
```
Comment (hodgestar) [inline docs/example.rst:34]: Are we missing the `[=>]` to indicate the these are sequences?

Comment (hodgestar) [inline docs/example.rst:34]: Ah, I see the type is now `symbol` and is picked up automatically by the parser.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**